### PR TITLE
Release notes for 1.6.0.

### PR DESCRIPTION
**Pull request by AveshCSingh**
Adds release notes for MLflow 1.6.0.

Primary @smurching 
FYI @mateiz @stbof

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+
+Features:
+
+- File store optimizations (#2339, @jonas)
+- Add feature importance plot for the XGBoost flavor (#2337, @harupy)
+- New runs table column view (#2251, @Zangr)
+- Add EarlyStopping integration to TensorFlow.Keras autologging (#2301, @juntai-zheng)
+- EarlyStopping Callback support for Keras Autologging (#2219, @juntai-zheng)
+- Add autodetection of job environments to R client (#2272, @dbczumar)
+- Add autodetection of job environments to Java client (#2286, @dbczumar)
```
Comment (stbof) [inline CHANGELOG.rst]: Combine two autodetection bullets?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+- Add auto logging functionality for LightGBM flavor. (#2275, @harupy)
```
Comment (stbof) [inline CHANGELOG.rst]: auto logging  -> autologging
Combine two autologging features: LightGBM and XGBoost

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
+- Python implementation of Spark datasource autologging (#2220, @smurching)
+- Add autologging functionality for XGBoost flavor (#2238, @harupy)
+- Reactivate Large Docker Tests (#2256, @andychow-db)
+- Add contour plot to the run comparsion page (#2225, @harupy)
+
+Bug fixes and documentation updates:
+
+- Escape % in db_url (#2289, @hershaw)
+- Fixed model docker image building on docker (#2257, @jahas)
```
Comment (stbof) [inline CHANGELOG.rst]: docker -> Docker

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,5 +1,38 @@
...
+- Solve race condition for directory creation (#2246, @avaucher)
+- Documentation for the SQL Server plugin (#2320, @avflor)
+- Documentation: Add help file for R package (#2259, @lorenzwalthert)
+- Adds an example of querying for the best model (#2313, @AveshCSingh)
+- Document MLflow plugin system (#2270, @smurching)
+- Regenerate R docs with up-to-date version of roxygen2 (#2258, @lorenzwalthert)
+- Fix typo in  documentation (#2261, @smith-kyle)
```
Comment (stbof) [inline CHANGELOG.rst]: Combine all the new documentation bullets. Don't report a typo fix.

**Comment by AveshCSingh** [comment]
Folks I realize this is the SWE equivalent of "dog ate my homework", but I accidentally did not push my local branch up. Sorry for sending out the sloppy changelog for review.

@stbof many of your suggestions were still relevant, and I've implemented them.

@mateiz still took your advice to make this more of a high level summary for people deciding whether to upgrade

File: CHANGELOG.rst [hunk 5]
```diff
@@ -1,5 +1,30 @@
...
 =========
+1.6.0 (2020-01-29)
+-----------------------
+MLflow 1.6.0 includes several new features, including a better runs table interface and a utility for easier parameter tuning. It also implements a long-awaited fix allowing @ symbols in URLs, and adds documentation of MLflow plugins, the Search API, and more. A complete list is below.
+
+Features:
+
+- Adds a new runs table column view based on `ag-grid` which adds functionality for nested runs, serverside sorting, column reordering, highlighting, and more. (#2251, @Zangr)
+- Adds contour plot to the run comparsion page to better support parameter tuning (#2225, @harupy)
+- Supports EarlyStopping callback in Keras autologging. When EarlyStopping.restore_best_weights==True, MLflow will now surface metrics of the restored model, rather than the model at the end of training. (#2301, #2219, @juntai-zheng)
```
Comment (AveshCSingh) [inline CHANGELOG.rst]: @smurching Do you think this description is accurate?

**Review by smurching — APPROVED** [review]
LGTM

**Comment by AveshCSingh** [comment]
One test failed in Travis:
tests/spark_autologging/test_spark_datasource_autologging_unit.py

This is a flakey test. It passed on an earlier commit in this PR:
https://travis-ci.org/mlflow/mlflow/builds/643579055?utm_medium=notification&utm_source=github_status

The only changes since that commit are textual, so I'm comfortable merging the PR.

**Pull request merged**